rero_ils/modules/organisations/jsonschemas/organisations/organisation-v0.0.1.json

```diff
@@ -40,6 +40,12 @@
       "description": "Code of the organisation.",
       "type": "string",
       "minLength": 2
+    },
+    "default_currency": {
```

Comment:
Maybe we have to say it is **mandatory**?

Reply:
You're right !
I think that `default_currency` should be a required field for each organization.
I will change the `json_schema` in that way.